evergreen-ci/spruce
# EVG-16195: Disable github fields when collision is found

EVG-16195

### Description 
- Disable PR Testing, Commit Checks, and Commit Queue if conflicts are found with other projects

### Screenshots
<img width="1478" alt="image" src="https://user-images.githubusercontent.com/9298431/160130782-b79b9b36-620b-4225-bf26-973f062fbb46.png">

### Testing 
To introduce conflicts between two projects (`evergreen` and `spruce`) locally:
1. Make sure you are running the latest `main` branch of evergreen
2. Visit http://localhost:3000/project/spruce/settings/general
3. Change the Repository name to `evergreen` and save
4. On the same page after saving, click "Attach to Current Repo" and confirm
5. Visit http://localhost:3000/project/evergreen/settings/github-commitqueue; you should see conflicts with Spruce's settings
